### ISSUE TITLE
Module `AccessibilityInfo` does not exist in the Haste module map

### ISSUE THREAD

**Issue by git-single-lee** [facebook/react-native#28479]
Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:183:15)
    at ResolutionRequest.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph.js:273:16)
    at Object.resolve (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/lib/transformHelpers.js:261:42)
    at dependencies.map.result (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:391:31)
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:388:18)
    at /Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:261:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
 BUNDLE  [dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1), failed.

**Comment by git-single-lee** [comment]
info 
  React Native Environment Info:
    System:
      OS: macOS 10.15.3
      CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
      Memory: 5.54 GB / 32.00 GB
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.4.0 - /usr/local/bin/node
      Yarn: 1.19.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 23, 25, 26, 27, 28, 29
        Build Tools: 25.0.0, 26.0.2, 26.0.3, 27.0.3, 28.0.2, 28.0.3, 29.0.0, 29.0.2
        System Images: android-27 | Google Play Intel x86 Atom
    IDEs:
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      react: ^16.8.3 => 16.8.3 
      react-native: ^0.59.10 => 0.59.10 
    npmGlobalPackages:
      react-native-cli: 2.0.1

**Comment by git-single-lee** [comment]
> ⚠️	Missing Environment Information
> ℹ️	Your issue may be missing information about your development environment. You can obtain the missing information by running `react-native info` in a console.

info 
  React Native Environment Info:
    System:
      OS: macOS 10.15.3
      CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
      Memory: 5.54 GB / 32.00 GB
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.4.0 - /usr/local/bin/node
      Yarn: 1.19.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 23, 25, 26, 27, 28, 29
        Build Tools: 25.0.0, 26.0.2, 26.0.3, 27.0.3, 28.0.2, 28.0.3, 29.0.0, 29.0.2
        System Images: android-27 | Google Play Intel x86 Atom
    IDEs:
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      react: ^16.8.3 => 16.8.3 
      react-native: ^0.59.10 => 0.59.10 
    npmGlobalPackages:
      react-native-cli: 2.0.1

**Comment by IOSTimor** [comment]
> Loading dependency graph, done.
> error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map
> 
> This might be related to #4968
> To resolve try the following:
> 
> 1. Clear watchman watches: `watchman watch-del-all`.
> 2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
> 3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
> 4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
>    at ModuleResolver.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:183:15)
>    at ResolutionRequest.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
>    at DependencyGraph.resolveDependency (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/node-haste/DependencyGraph.js:273:16)
>    at Object.resolve (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/lib/transformHelpers.js:261:42)
>    at dependencies.map.result (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:391:31)
>    at Array.map ()
>    at resolveDependencies (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:388:18)
>    at /Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:261:33
>    at Generator.next ()
>    at asyncGeneratorStep (/Users/lishuaiqi820/codeToGit/goodHelper/hbs_phoenix_android/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
>    BUNDLE  [dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1), failed.

你是怎么解决这个问题的呢，大兄弟
